packages/react-components/src/components/kpi/kpi.spec.tsx

```diff
@@ -35,6 +36,6 @@ it('renders', async () => {
     `${DATA_STREAM.data[0].y} `
   );
   expect(screen.getByTestId('kpi-timestamp').textContent).toContain(
-    new Date(DATA_STREAM.data[0].x).toLocaleString()
```

Comment:
this will probs have some merge conflicts with my changes here https://github.com/awslabs/iot-app-kit/pull/2943 

probably worth merging this PR first and then i can rebase and add `formatDates` everywhere 😄